set.mm

```diff
@@ -504719,6 +504773,24 @@ its graph has a given second element (that is, function value).
     GDJCKADUGLMNUJUFUHUJUFUHOCUIFZUGUIJZOUKBCPQFULCGBSCGBRTUJUFUKUHULAUICUAAUIU
     GLUBMUCUD $.
 
+  ${
+    fzo0opth.1 $e |- ( ph -> M e. NN0 ) $.
+    fzo0opth.2 $e |- ( ph -> N e. NN0 ) $.
+    $( Equality for a half open integer range starting at zero is the same as
+       equality of its upper bound.  (Contributed by Thierry Arnoux,
```

Comment:
Maybe "analogous to ~ fzopth or ~ fzoopth." should be added, to make clear the suffix "opth" a little bit.

Reply:
Added in https://github.com/metamath/set.mm/pull/4855/commits/b5a4a71bfbaf9d6a91896c855adfba6f3b0db9c2